AppleALC/Info.plist

```diff
@@ -40,26 +57,28 @@
 	</dict>
 	<key>NSHumanReadableCopyright</key>
 	<string>Copyright © 2017 vit9696. All rights reserved.</string>
+	<key>OSBundleCompatibleVersion</key>
+	<string>1.0</string>
 	<key>OSBundleLibraries</key>
 	<dict>
+		<key>as.vit9696.Lilu</key>
+		<string>1.2.0</string>
+		<key>as.vit9696.PinConfigs</key>
```

Comment:
We actually no longer have PinConfigs I believe. Could you drop this since you are changing the code anyway?

Reply:
Done.